docs/sources/write/style-guide/write-for-developers/index.md

```diff
@@ -78,23 +78,135 @@ Additional tips:
 Readers of documentation typically skim through it to find code samples they can copy and paste and run as is.
 Because of this and whenever possible, provide production-ready examples.
 
-However, it isn't necessary for every code example to be runnable in production. Some code examples are written to illustrate a point so that the developer can learn how to do something similar on their own.
+However, it isn't necessary for every code example to be runnable in production.
+Some code examples are written to illustrate a point so that the developer can learn how to do something similar on their own.
+Such examples should be clearly marked as partial.
 
 When providing an example, give a written description. You can put it either in the body of the document or as explanatory comments within the example code.
 
 Remember the basic rule: explain _why_ your code does what it does, rather than describe _what_ it does.
 For an in-depth, external resource about writing developer documentation, see [_Docs for Developers_](https://docsfordevelopers.com/).
 
-### Formatting code examples
+### Format code examples
```

Comment:
I think starting these two formatting headings with the verb (especially since they aren't tasks) hides what we're trying to really get across here, which is one of these sections is about full code samples and one is about partial code samples. I've made the associated suggestion on line 137.
```suggestion
### Full code examples
```